src/main/resources/markdown/releases.md

```diff
@@ -42,18 +42,15 @@ Older releases are available.
 
 [Apache ZooKeeper 3.6.2 Source Release](https://www.apache.org/dyn/closer.lua/zookeeper/zookeeper-3.6.2/apache-zookeeper-3.6.2.tar.gz)([asc](https://downloads.apache.org/zookeeper/zookeeper-3.6.2/apache-zookeeper-3.6.2.tar.gz.asc), [sha512](https://downloads.apache.org/zookeeper/zookeeper-3.6.2/apache-zookeeper-3.6.2.tar.gz.sha512))
 
-### Apache ZooKeeper 3.6.1
-
-[Apache ZooKeeper 3.6.1](https://www.apache.org/dyn/closer.lua/zookeeper/zookeeper-3.6.1/apache-zookeeper-3.6.1-bin.tar.gz)([asc](https://downloads.apache.org/zookeeper/zookeeper-3.6.1/apache-zookeeper-3.6.1-bin.tar.gz.asc), [sha512](https://downloads.apache.org/zookeeper/zookeeper-3.6.1/apache-zookeeper-3.6.1-bin.tar.gz.sha512))
-
-[Apache ZooKeeper 3.6.1 Source Release](https://www.apache.org/dyn/closer.lua/zookeeper/zookeeper-3.6.1/apache-zookeeper-3.6.1.tar.gz)([asc](https://downloads.apache.org/zookeeper/zookeeper-3.6.1/apache-zookeeper-3.6.1.tar.gz.asc), [sha512](https://downloads.apache.org/zookeeper/zookeeper-3.6.1/apache-zookeeper-3.6.1.tar.gz.sha512))
+### Apache ZooKeeper 3.5.9
 
+ZooKeeper 3.5.9 is only downloadable from the archive as the 3.6 and 3.7 branches are now `stable` and `current`, respectively.
```

Comment:
>> the 3.6 and 3.7 branches are now `stable` and `current`, respectively.

This is correct.

But in the starting of this file it is written that "Apache ZooKeeper 3.7.0 is our latest stable release". is this statement just a carry forward from previous release or there is some thought behind declaring 3.7.0 as stable release

I think "Apache ZooKeeper 3.7.0 is our latest stable release" to be changed to "Apache ZooKeeper 3.7.0 is our current release" and later based on usage and feedback of 3.7.0 we can change 3.7.x as stable release.

Reply:
Good point.  That sentence was just carried forward when I switched the versions in the release notes, and I certainly agree that it is a bit early to declare 3.7 stable compared to 3.6.  How about the updated patch?  (It implements your suggestion, but I have also moved 3.6 above the separator line.)

Reply:
We have currently 3 active branches/release lines:
- 3.5
- 3.6
- 3.7

I am not sure we should say much here.

we just have to give the links to binary/sources for the latest release of 3.5,3.6 and 3.7, so 3.5.9, 3.6.2, 3.7.0

Reply:
I am not sure we should delete 3.5.9 from the "dist", it is an active release line.
if we want to change this fact we should open a discussion on dev@

Good catch and good points @ztzg @arshadmohammad

Reply:
Well, I removed 3.5.9 from the "dist" (as per the instructions, not because I have something against it :) when I prepared 3.7.0.  I guess I should restore it in SVN, which will in turn cause it to be mirrored again?